src/types.rs

```diff
@@ -226,18 +202,16 @@ fn ns_hash_from_b64(input: &str) -> NamespacedHash {
 #[cfg(test)]
 mod tests {
 
-    use nmt_rs::{NamespaceProof, NamespacedSha2Hasher};
-
-    use crate::verifier::ROLLUP_NAMESPACE;
+    // use nmt_rs::{NamespaceProof, NamespacedSha2Hasher};
 
-    use super::{ns_hash_from_b64, RpcNamespacedSharesResponse};
+    // use super::{ns_hash_from_b64, RpcNamespacedSharesResponse};
```

Comment:
Maybe delete commented code, if we don't need it.

Reply:
I think Ieft it since we plan to re-enable this test once a Celestia bug is fixed, and we'll need this stuff to enable the test.